app/controllers/providers/allocations_controller.rb

```diff
@@ -27,6 +28,8 @@ def index
 
     def repeat_request; end
```

Comment:
Given we have defined `edit_repeat_request` I wonder if we should rename this action (and template) `new_edit_request` to appease the Rails-ey consistency

Reply:
Agreed `new_` would be more explicit here. I'm not sure we need the `edit_repeat_request` version though. We know the state of the `Allocation` (as per comment above) so `edit` should be fine and avoid adding too many 'non-standard' actions.